_articles/assets/models/adding-hitbox-to-models-without-a-bone.md

```diff
@@ -69,4 +68,4 @@ https://puu.sh/lLIJ3/bed9e07b6f.jpg
 
```

Comment:
hitbox vmdl URL in this article needs to be updated because it was moved from resources to static